### ISSUE TITLE
Failure to execute on debian based systems

### ISSUE THREAD

**Issue by TaiSHiNet** [holland-backup/holland#313]
Systems tested:
* Debian 9
* Ubuntu 18.04.5

Package versions:
* holland 1.2.2-0
* python3-pymysql 0.7.10-1 (Debian)
* python3-pymysql 0.8.0-1 (Ubuntu)

Output:
```# holland bk -n
Holland 1.2.2 started with pid 6338
--- Starting dry run ---
Creating backup path /var/spool/holland/default/20200914_103628
Backup failed: Could not find dependency 'The 'python3-pymysql' distribution was not found and is required by holland.lib.mysql'
--- Ending dry run ---```

Also, python3-pymysql stopped being a dependency in the debian package (which led to finding this issue). Previous working version was `1.2.1-0`

**Comment by soulen3** [comment]
This is expected behavior, and is mostly due to my lack on experience with Debain packaging.

This is documented in the install instructions
https://hollandbackup.org/INSTALL.html

**Comment by TaiSHiNet** [comment]
@soulen3 the packages are installed (as mentioned in the guide) yet holland is complaining they aren't. This was working on previous version.

**Comment by soulen3** [comment]
I'm not really sure what would have caused this. There weren't any changes to the holland.lib.mysql module between 1.2.1 and 1.2.2.  This doesn't sound like an issue with the deb package, as that hasn't change and holland is installed.

**Comment by soulen3** [comment]
I'm not sure what happened yet, but a work around is to edit `/usr/lib/python3/dist-packages/holland.lib.mysql-1.2.2.egg-info/requires.txt`

Change `python3-pymysql` to `pymysql`

**Comment by TaiSHiNet** [comment]
Workaround works around perfectly! Thanks!

**Comment by soulen3** [comment]
So it was an issue in the deb. I've corrected it, and created a new release for 1.2.2.  Please let me know if you're still having problems with 1.2.2-1.

Thanks for reporting this error.

**Comment by TaiSHiNet** [comment]
Perfect!
Tested:
* Upgrading and attempting to remove python3-pymysql (would remove holland)
* Removing python3-pymysql and upgrading holland would install python3-pymysql
* Removing the workaround (prior to upgrading) and tested backups

Everything works as expected, thanks a lot for the quick fix!
